test/load/allocation/grpc/README.md

```diff
@@ -74,3 +74,157 @@ You can use environment variables overwrite defaults. To run only a single run o
 ```
 TESTRUNSCOUNT=1 ./runAllocation.sh 40 10
 ```
+
+
+# Running Scenaio tests
+
+The scenario test allows you to generate a variable number of allocations to
+your cluster over time, simulating a game where clients arrive in an unsteady
+pattern.
+
+## Kubernetes Cluster Setup
+
+For the scenario test to achieve high throughput, you can create multiple groups
+of nodes in your cluster. During testing (on GKE), we created a node pool for
+the Kubernetes system components (such as the metrics server and dns servers), a
+node pool for the Agones system components (as recommended in the installation
+guide), and a node pool for the game servers.
+
+On GKE, to restrict the Kubernetes system components to their own set of nodes,
+you can create a node pool with the taint
+`components.gke.io/gke-managed-components=true:NoExecute`.
+
+To prevent the Kubernetes system components from running on the game servers
+node pool, that node pool was created with the tain
+`scenario-test.io/game-servers=true:NoExecute`
+and the Agones system node pool used the normal taint
+`agones.dev/agones-system=true:NoExecute`.
+
+In addition, the GKE cluster was configured as a regional cluster to ensure high
+availability of the cluster control plane.
+
+## Agones Modifications
+
+For the scenario tests, we modified the Agones installation in a number of ways.
+
+First, we made sure that the Agones pods would _only_ run in the Agones node
+pool by changing the node affinity in the deployments for the controller,
+allocator service, and ping service to
+`requiredDuringSchedulingIgnoredDuringExecution`.
+
+We also increased the resources for the controller and allocator service pods,
+and made sure to specify both requests and limits to ensure that the pods were
+given the highest quality of service.
+
+
+## Fleet Setting
+
+We used the following fleet configuration:
+
+```
+apiVersion: "agones.dev/v1"
+kind: Fleet
+metadata:
+  name: scenario-test
+spec:
+  replicas: 10
+  template:
+    metadata:
+      labels:
+        gameName: simple-game-server
+    spec:
+      ports:
+      - containerPort: 7654
+        name: default
+      health:
+        initialDelaySeconds: 30
+        periodSeconds: 60
+      template:
+        spec:
+          tolerations:
+          - effect: NoExecute
+            key: scenario-test.io/game-servers
+            operator: Equal
+            value: 'true'
+          containers:
+          - name: simple-game-server
+            image: gcr.io/agones-images/simple-game-server:0.9
```

Comment:
nit:I think we have 0.10 and we recently used that one.

Reply:
The most recent one I ran was 0.9 but it looks like we have published a 0.10. I'll switch it to the most recent version.